### PR TITLE
Update ECS.NET documentation on elastic.co

### PR DESCRIPTION
**Pull request by Mpdreamz**
Fixes: https://github.com/elastic/ecs-dotnet/issues/202

Preview: https://ecs-dotnet_301.docs-preview.app.elstc.co/guide/en/ecs-logging/dotnet/current/intro.html

### CODE REVIEW

**Comment by Mpdreamz** [comment]
test

**Comment by Mpdreamz** [comment]
run elasticsearch-ci/docs

File: docs/data-shippers/extensions-logging.asciidoc [hunk 1]
```diff
@@ -0,0 +1,444 @@
...
+
+|===
+|Setting |Type |Description 
+
+|IncludeHost |boolean |Default `true`; set to `false` to disable logging host values. 
+|IncludeProcess |boolean |Default `true`; set to `false` to disable logging process values. 
+|IncludeScopes |boolean |Default `true`; set to `false` to disable logging scope values. 
+|IncludeUser |boolean |Default `true`; set to `false` to disable logging user details. 
+|Index |format |Format string used to generate the Elasticsearch `index`, using the current timestamp. Default is `dotnet-{0:yyyy.MM.dd}`. 
+|IndexOffset |timespan |Override to set the offset used to generate the `index`. Default value is `null`, which uses the system local offset; use `&quot;00:00&quot;` for UTC.
```
Comment (bmorelli25) [inline docs/data-shippers/extensions-logging.asciidoc:107]:
```suggestion
|IndexOffset |timespan |Override to set the offset used to generate the `index`. Default value is `null`, which uses the system local offset; use `"00:00&"` for UTC. 
```

File: docs/data-shippers/extensions-logging.asciidoc [hunk 2]
```diff
@@ -0,0 +1,444 @@
...
+
+|IncludeHost |boolean |Default `true`; set to `false` to disable logging host values. 
+|IncludeProcess |boolean |Default `true`; set to `false` to disable logging process values. 
+|IncludeScopes |boolean |Default `true`; set to `false` to disable logging scope values. 
+|IncludeUser |boolean |Default `true`; set to `false` to disable logging user details. 
+|Index |format |Format string used to generate the Elasticsearch `index`, using the current timestamp. Default is `dotnet-{0:yyyy.MM.dd}`. 
+|IndexOffset |timespan |Override to set the offset used to generate the `index`. Default value is `null`, which uses the system local offset; use `&quot;00:00&quot;` for UTC. 
+|IsEnabled |boolean |Default `true`; set to `false` to disable the logger. 
+|ListSeparator |string |Separator to use for `IEnumerable` in `labels.*` values. Default is `&quot;, &quot;`. 
+|Tags |array |Additional tags to include in the message. Useful to specify the environment or other details, e.g. `[ &quot;Staging&quot;, &quot;Priority&quot; ]`
```
Comment (bmorelli25) [inline docs/data-shippers/extensions-logging.asciidoc:110]:
```suggestion
|ListSeparator |string |Separator to use for `IEnumerable` in `labels.*` values. Default is `", "`. 
|Tags |array |Additional tags to include in the message. Useful to specify the environment or other details, e.g. `[ "Staging", "Priority"]` 
```

File: docs/data-shippers/extensions-logging.asciidoc [hunk 3]
```diff
@@ -0,0 +1,444 @@
...
+
+The ShipTo settings can have the following properties, depending on the type of connection pool.
+
+|===
+|Setting |Type |Description 
+
+|ApiKey |string |API Key, where connection pool type is Cloud, and authenticating via API Key. 
+|CloudId |string |Cloud ID, where connection pool type is Cloud. 
+|NodePoolType |enum |Default is `Singlenode`, or `Sniffing` for multiple nodes, or `Cloud` if `CloudId` is provided. Other supported values are `Static` or `Sticky`. 
+|NodeUris |array |URI(s) of the Elasticsearch nodes to connect to. Default is a single node `[ &quot;http://localhost:9200&quot; ]`
```
Comment (bmorelli25) [inline docs/data-shippers/extensions-logging.asciidoc:121]:
```suggestion
|NodeUris |array |URI(s) of the Elasticsearch nodes to connect to. Default is a single node `[ "http://localhost:9200" ]` 
```

File: docs/data-shippers/extensions-logging.asciidoc [hunk 4]
```diff
@@ -0,0 +1,444 @@
...
+----
+
+==== Standard Fields
+
+|===
+|Field |Type |Description 
+
+|@timestamp |date |`DateTimeOffset` when the message was logged, including local offset. 
+|message |string |The formatted log message and arguments. 
+|tags |array |Custom tags from configuration, e.g. `[ &quot;Staging&quot;, &quot;Priority&quot; ]`. Can have multiple values.
```
Comment (bmorelli25) [inline docs/data-shippers/extensions-logging.asciidoc:259]:
```suggestion
|tags |array |Custom tags from configuration, e.g. `[ "Staging", "Priority" ]`. Can have multiple values. 
```

File: docs/data-shippers/extensions-logging.asciidoc [hunk 5]
```diff
@@ -0,0 +1,444 @@
...
+
+|===
+|Type |Formatting 
+
+|byte |Hex, e.g. "9A" 
+|byte[] |Prefixed hex, e.g. "0x12789AF0" 
+|DateTimeOffset |ISO format, e.g. "2020-01-02T03:04:05.000000+06:00" 
+|DateTime |In most cases `DateTimeOffset` should be used instead (1). Where `DateTime` is used for date only (with no time component), it is formatted as a date, e.g. "2020-01-02". If it has a time component, the roundtrip ("o") format is used. 
+|IEnumerable |Values separated by ", " (configurable) 
+|IDictionary&lt;string, object&gt; |A string containing key value pairs, e.g. `token=&quot;0x12789AF0&quot; count=&quot;5&quot;`
```
Comment (bmorelli25) [inline docs/data-shippers/extensions-logging.asciidoc:331]:
```suggestion
|IDictionary&lt;string, object&gt; |A string containing key value pairs, e.g. `token="0x12789AF0" count="5"` 
```

**Review by bmorelli25 — COMMENTED** [review]
This is a great addition to the docs. I have some suggestions for you to consider.

File: docs/intro.asciidoc [hunk 6]
```diff
@@ -7,4 +7,76 @@ They make it easy to format your logs into ECS-compatible JSON.
...
+All the data shippers utilize <<ecs-ingest-channels, `Elastic.Ingest.Elasticsearch.CommonSchema`>> to send events to Elasticsearch.
+
+[float]
+=== Enrichers 
+
+Enrichers can be installed next to `Log Formatters` and `Data Shippers` to automatically
+enrich the ECS json that gets produced.
+
+* `Elastic.Apm.NLog`
+* `Elastic.Apm.SerilogEnricher`
```
[the remark this answers is not in the document]
Comment (Mpdreamz) [reply docs/intro.asciidoc:60]: Complete oversight on my part, thanks for catching it!

**Comment by Mpdreamz** [comment]
Thank you for the extensive review @bmorelli25 !

**Pull request merged**